src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -413,6 +413,7 @@ def post_request(
             alias,
             uri,
             data=None,
+            json=None,
```

Comment:
Late to bring this up @bulkan, but inserting new arguments like this is not backwards compatible. If someone was calling `Post Request` with positional arguments, the value for `params` became the value for `json`. Adding `json` to the end would have avoided this. This is a great feature to have btw. When going from Python usage of requests, have been missing convenience of `json` arg.